build.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
+# Required Environment Variables (values here are from stagenet)
+CHAIN_ID=8081
+NEXT_PUBLIC_CHAIN_ID=${CHAIN_ID}
+APP_MONITOR="34.16.2.42"
+RPC_SERVER_URL="https://api-stagenet.shardeum.org"
+EXISTING_ARCHIVERS='[{"ip":"34.57.177.170","port":4000,"publicKey":"d831bb7c09db45d47338af23ab50cac5d29ef8f3a2cd274dd741370aa472d6c1"},{"ip":"34.73.104.156","port":4000,"publicKey":"37d162292c068030bbde55ee8ac777ad08f443b9bba0b0064e4919345d43727a"},{"ip":"35.230.76.119","port":4000,"publicKey":"d8475a2d2110becb2d031085a2915956239dafcafd2d57f5468a187bb7235dd7"}]'
+NEXT_PUBLIC_RPC_URL="https://api-stagenet.shardeum.org"
+NEXT_PUBLIC_EXPLORER_URL="https://explorer-stagenet.shardeum.org"
+minNodes=360
+baselineNodes=360
+nodesPerConsensusGroup=128
+maxNodes=1280
+enableProblematicNodeRemoval=true
+enableProblematicNodeRemovalOnCycle=0
+flexibleRotationDelta=4
+VALIDATOR_BRANCH=mainnet-launch
+CLI_BRANCH=mainnet-launch
+GUI_BRANCH=mainnet-launch
```

Comment:
We should let this be testnet configs and not stagenet configs by default.
cc @chrischabot

Reply:
fixed in https://github.com/shardeum/shardeum-validator/pull/16/commits/ba4ef91e9ad64e3db0f18045e6ead4925ff36bc3